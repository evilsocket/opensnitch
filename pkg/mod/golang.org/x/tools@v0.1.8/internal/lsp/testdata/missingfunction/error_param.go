package missingfunction

func errorParam() {
	var err error
	undefinedErrorParam(err) //@suggestedfix("undefinedErrorParam", "quickfix")
}
