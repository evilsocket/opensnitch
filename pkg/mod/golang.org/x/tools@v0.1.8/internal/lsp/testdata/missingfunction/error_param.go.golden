-- suggestedfix_error_param_5_2 --
package missingfunction

func errorParam() {
	var err error
	undefinedErrorParam(err) //@suggestedfix("undefinedErrorParam", "quickfix")
}

func undefinedErrorParam(err error) {
	panic("unimplemented")
}

